stable/selenium/templates/hub-deployment.yaml

```diff
@@ -88,4 +88,16 @@ spec:
 {{ toYaml .Values.hub.nodeSelector | indent 8  }}
 {{- else if .Values.global.nodeSelector }}
 {{ toYaml .Values.global.nodeSelector | indent 8  }}
+{{- end }}
+      affinity:
+{{- if .Values.chromeDebug.affinity }}
```

Comment:
Same mistake as above, .Values.chromeDebug.affinity should be replaced with .Values.hub.affinity (same typo below).

Reply:
thanks for review, it's fixed.